<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XmlPresentationWidget</class>
 <widget class="QWidget" name="XmlPresentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="5">
    <widget class="XmlPresentationTreeView" name="m_presentationTreeView">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QProgressBar" name="m_presentationProgressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>163412</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Filtre :</string>
     </property>
     <property name="buddy">
      <cstring>m_filtreLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="XinxLineEdit" name="m_filtreLineEdit">
     <property name="whatsThis">
      <string>Filter to use in the view. &lt;b&gt;XINX&lt;/b&gt; searches the node where the name matches the filter (literal or regular expression).</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="m_filterComboBox">
     <property name="whatsThis">
      <string>If &lt;i&gt;Sub-element&lt;/i&gt; is selected when an element is found, all sub-elements are showed. &lt;br/&gt;
If &lt;i&gt;None&lt;/i&gt; is selected only matched elements is showed.</string>
     </property>
     <item>
      <property name="text">
       <string>Sub-element</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XinxLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">xinxlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>XmlPresentationTreeView</class>
   <extends>QTreeView</extends>
   <header>docks/datastream/xmlpresentationtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
